src/encoded/schemas/biosample.json

```diff
@@ -0,0 +1,114 @@
+{
+    "title": "Biosample",
+    "id": "/profiles/biosample.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "required": [
+        "category"
+    ],
+    "anyOf": [
+        {
+            "required": [
+                "submission_centers",
+                "submitter_id"
+            ]
+        },
+        {
+            "required": [
+                "consortia"
+            ]
+        }
+    ],
+    "identifyingProperties": [
+        "accession",
+        "aliases",
+        "center_submitter_ids",
+        "identifier",
+        "uuid"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/aliases"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/category"
+        },
+        {
+            "$ref": "mixins.json#/description"
+        },
+        {
+            "$ref": "mixins.json#/identifier"
+        },
+        {
+            "$ref": "mixins.json#/modified"
+        },
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/status"
+        },
+        {
+            "$ref": "mixins.json#/submitted"
+        },
+        {
+            "$ref": "mixins.json#/submitter_id"
+        },
+        {
+            "$ref": "mixins.json#/tags"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        }
+    ],
+    "type": "object",
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "accession": {
+            "accessionType": "BS"
+        },
+        "category": {
+            "enum": [
+                "Bulk Tissue",
+                "Single Cell"
+            ]
+        },
+        "biosources": {
+            "title": "Biosources",
+            "type": "array",
+            "uniqueItems": true,
+            "minItems": 1,
+            "items": {
+                "title": "Biosource",
+                "type": "string",
+                "linkTo": "Biosource"
+            }
+        },
```

Comment:
Could a biosample come from multiple biosources? How would that work? Mixed sample?